processing/src/main/java/org/apache/druid/segment/index/SimpleImmutableBitmapIndex.java

```diff
@@ -35,6 +35,12 @@ public SimpleImmutableBitmapIndex(ImmutableBitmap bitmap)
     this.bitmap = bitmap;
   }
 
+  @Override
+  public int estimatedComputeCost()
+  {
+    return 0;
```

Comment:
Right this index seems mainly for null index, so it's just 1 bitmap with no union. When I looked up `forValue` seems like it's possible there're two bitmaps (one for the value and one for null) with one union. That's why i decided 0 for this, and 1 for other `SimpleBitmapIndex` instances. I feel `SimpleImmutableBitmapIndex` is slightly cheaper since no binary search for dictionary and no bitmap union.